scripts/globals/weaponskills.lua

```diff
@@ -1357,9 +1358,30 @@ function handleWSGorgetBelt(attacker)
     local accBonus = 0
 
     if attacker:getObjType() == xi.objType.PC then
-        -- TODO: Get these out of itemid checks when possible.
-        local elementalGorget = { 15495, 15498, 15500, 15497, 15496, 15499, 15501, 15502 }
-        local elementalBelt =   { 11755, 11758, 11760, 11757, 11756, 11759, 11761, 11762 }
+        local elementalGorget = -- Ordered by element correctly.
```

Comment:
request: change comment to `-- Ordered by element, TDODO: mods/latents instead of items`

Even if the item ID is variablized, its still using item ids and someday it'll get changed not to as long as something reminds a future editor of that

Reply:
Done